builtin/credential/ldap/path_config.go

```diff
@@ -367,131 +358,6 @@ func (b *backend) pathConfigWrite(ctx context.Context, req *logical.Request, d *
 	return nil, nil
 }
 
-type ConfigEntry struct {
-	logger        log.Logger
-	Url           string `json:"url"`
-	UserDN        string `json:"userdn"`
-	GroupDN       string `json:"groupdn"`
-	GroupFilter   string `json:"groupfilter"`
-	GroupAttr     string `json:"groupattr"`
-	UPNDomain     string `json:"upndomain"`
-	UserAttr      string `json:"userattr"`
-	Certificate   string `json:"certificate"`
-	InsecureTLS   bool   `json:"insecure_tls"`
-	StartTLS      bool   `json:"starttls"`
-	BindDN        string `json:"binddn"`
-	BindPassword  string `json:"bindpass"`
-	DenyNullBind  bool   `json:"deny_null_bind"`
-	DiscoverDN    bool   `json:"discoverdn"`
-	TLSMinVersion string `json:"tls_min_version"`
-	TLSMaxVersion string `json:"tls_max_version"`
-
-	// This json tag deviates from snake case because there was a past issue
-	// where the tag was being ignored, causing it to be jsonified as "CaseSensitiveNames".
-	// To continue reading in users' previously stored values,
-	// we chose to carry that forward.
-	CaseSensitiveNames *bool `json:"CaseSensitiveNames,omitempty"`
-}
-
-func (c *ConfigEntry) GetTLSConfig(host string) (*tls.Config, error) {
```

Comment:
This was moved with no change. There's no external call so I put it on the config, and it didn't need to be exported anymore.